tests/integration/test_default_compression_codec/test.py

```diff
@@ -436,7 +436,7 @@ def test_default_codec_version_update(start_cluster):
     old_version = node3.query("SELECT version()")
     node3.restart_with_latest_version(fix_metadata=True)
     new_version = node3.query("SELECT version()")
-    logging.debug(f"Updated from {old_version} to {new_version}")
```

Comment:
This kind of changes are making the code less readable. Why should we care about `lazy evaluation of the string` or untrusted strings in test files?

I'd rather have more f-strings, not less.

Reply:
> Why should we care about lazy evaluation of the string or untrusted strings in test files?

It might (and will) improve the tests performance, as not needed strings will not be evaluated. I don't have a strong opinion about that though

Reply:
I find it extremely hard to believe that changing f-strings to '%' can have any impact on code that's testing an external database. The time spent logging in Python is negligible, so even if we removed all logging I wouldn't expect any change in performance.

Reply:
> It might (and will) improve the tests performance

Do you have some benchmark results demonstrating the performance improvement? 

Well, we can run the CI and check `check_duration_ms` and `test_duration_ms` in the [CI DB](https://play.clickhouse.com/play?user=play#ZGVzYyBjaGVja3MK), but I doubt it will show any difference

Reply:
If you find it harder to read this way, please feel free to close the PR.